content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -38,6 +38,35 @@ that provides the expected symbols.
 
 <!-- steps -->
 
+## Check your OS kernel version
+
+{{< tabs name="container_runtime" >}}
+{{% tab name="Linux" %}}
+
+* The Linux kernel must be 4.19 or newer, 5.x, or 6.x.
+* You can get the kernel version using the command `uname -r`
+
+{{% /tab %}}
+
+{{% tab name="Windows" %}}
+
+* The Windows kernel version must be 10.0.14393 or newer.
+* You can get the kernel version (also called the OS version) using the command `systeminfo`
```

Comment:
a similar case here.

we can link to these 2 pages:
- https://learn.microsoft.com/en-us/windows/release-health/windows-server-release-info
- https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes/#before-you-begin
instead of hardcoding the version.

Reply:
Fxied